Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift

```diff
@@ -211,14 +211,22 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
 
         createFormRow(with: &rows, item: product.categoriesTags?.map({ (categoryTag: String) -> NSAttributedString in
             if let category = dataManager.category(forTag: categoryTag) {
-                if let name = Tag.choose(inTags: Array(category.names)) {
+                if let name = Tag.choose(inTags: Array(category.names), forLanguageCode: Bundle.main.currentLocalization) {
                     return NSAttributedString(string: name.value, attributes: [NSAttributedString.Key.link: OFFUrlsHelper.url(forCategory: category)])
                 }
             }
             return NSAttributedString(string: categoryTag)
         }), label: InfoRowKey.categories.localizedString)
 
-        createFormRow(with: &rows, item: product.labels, label: InfoRowKey.labels.localizedString)
+        createFormRow(with: &rows, item: product.labelsTags?.map({ (labelTag: String) -> NSAttributedString in
+            if let label = dataManager.label(forTag: labelTag) {
+                if let name = Tag.choose(inTags: Array(label.names), forLanguageCode: Bundle.main.currentLocalization) {
```

Comment:
@teolemon 
Here is where I added the code to translate Labels by fetching the corresponding taxonomy of the Label.